modin/pandas/test/test_series.py

```diff
@@ -4076,7 +3929,7 @@ def test_str_removesuffix(data):
 
 
 @pytest.mark.parametrize("data", test_string_data_values, ids=test_string_data_keys)
-@pytest.mark.parametrize("width", int_arg_values, ids=int_arg_keys)
+@pytest.mark.parametrize("width", [-1, 0, 5])
```

Comment:
There are an excessive number of parameterization options; we can reduce their number without changing the coverage.